index.html

```diff
@@ -184,6 +187,19 @@ <h2> About <br> Myself </h2>
         </ul>
       </div>
     </section>
+
+    <footer class="contact rounded-corner shapes-bg-2">
+      <h2 id="contact">Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+      </p>
+      <form id="contact-form" action="https://formspree.io/f/xgedkalg" method="POST">
+        <input type="text" name="name" id="name" placeholder="Enter you name" aria-label="User name" value="Yeremias">
+        <input type="email" name="email" id="email" placeholder="your.email@domain.com" value="natajayanj14@gmail.com">
+        <textarea name="message" id="message" maxlength="500" placeholder="Write your message here"></textarea>
+      </form>
+      <button class="btn" form="contact-form" type="submit">Get In Touch</button>
+    </footer>
```

Comment:
- [ ] There is an extra space to the right of your page. Kindly remove this as it is not a good user experience. 

![image](https://user-images.githubusercontent.com/43995830/154155806-fc4eecd3-32f2-4cf8-9e72-5ab6df3d337e.png)

Reply:
Thank you! Using the browser's DevTools to simulate the mobile is certainly a better strategy (thanks @wuletawwonte for your suggestion)